docker/resource-provider/Dockerfile

```diff
@@ -33,24 +33,13 @@ RUN mv lilypad /usr/local/bin
 # Install necessary dependencies
 RUN apt update && apt install -y wget bash curl && apt clean
 
-# Install Bacalhau
-RUN cd /tmp && \
-    wget https://github.com/bacalhau-project/bacalhau/releases/download/v1.3.2/bacalhau_v1.3.2_linux_amd64.tar.gz && \
-    tar xfv bacalhau_v1.3.2_linux_amd64.tar.gz && \
-    mv bacalhau /usr/local/bin/bacalhau && \
-    rm bacalhau_v1.3.2_linux_amd64.tar.gz
-
```

Comment:
Using the http API means we no longer need to *also* have the bacalhau CLI installed here